src/mscorlib/src/System/Enum.cs

```diff
@@ -844,20 +900,23 @@ public int CompareTo(Object target)
 
         #region Public Methods
         public String ToString(String format) {
+            char formatCh;
             if (format == null || format.Length == 0)
-                format = "G";
+                formatCh = 'G';
+            else
```

Comment:
49% of the time is spent in `System.Reflection.CustomAttributeData.GetCustomAttributeRecords`
43% in `System.RuntimeType.IsSubclassOf`